app/presenters/publishing_api_presenters/publication.rb

```diff
@@ -0,0 +1,69 @@
+require_relative "../publishing_api_presenters"
+
+class PublishingApiPresenters::Publication < PublishingApiPresenters::Edition
+  include PublishingApiPresenters::WithdrawingHelper
+  include PublishingApiPresenters::PoliticalHelper
+  include PublishingApiPresenters::ApplicabilityHelper
+
+  def links
+    extract_links([
+      :lead_organisations,
```

Comment:
As mentioned elsewhere, `lead_organisations` is no longer necessary to sent seperately.